app/controllers/retirement_controller.rb

```diff
@@ -32,9 +32,10 @@ def destroy_subscription
     Subscription.new.destroy(current_user.subscription_id) if current_user.subscription_id
   end
 
-  def notify_to_admins
+  def notify_to_admins(current_user)
```

Comment:
@taka61 さん、
Approvedしましたが、その後、もう一度考えて、以下のコードがいかがでしょうか。
```suggestion
  def notify_to_admins
```